packages/app/src/components/nominationBanner/style.css

```diff
@@ -37,9 +37,9 @@
 }
 
 .secondary-dark {
-  color: var(--secondary-dark);
+  font-size: 1.1rem !important;
```

Comment:
Why do you need the `!important` here? I would avoid using this as it breaks the cascading styles flow. Let me know if you want to work together to work to remove this.